src/xunit.console.netcore/Visitors/StandardOutputVisitor.cs

```diff
@@ -13,6 +16,12 @@ public class StandardOutputVisitor : XmlTestExecutionVisitor
         readonly ConcurrentDictionary<string, ExecutionSummary> completionMessages;
         readonly string defaultDirectory;
         readonly bool showProgress;
+        readonly Stopwatch clock;
+        private ConcurrentDictionary<String, long> runningTests;
+        private Thread watcher;
+        readonly int longTestMaxMiliseconds = 1_000 * 60 * 5;
```

Comment:
both this and below should be const (readonly means "modified only in constructor")